hvplot/tests/testoperations.py

```diff
@@ -194,6 +195,11 @@ def test_datashade_rescale_discrete_levels_default_True(self):
         actual = plot.callback.inputs[0].callback.operation.p['rescale_discrete_levels']
         assert actual is expected
 
+    def test_rasterize_by(self):
```

Comment:
This should also have a version check, but let us ignore it if all tests pass.

Reply:
Yep for us it works well but sometimes distros that re-package our tools complain about this kind of stuff (and probably just patch things).

Reply:
Ok I'll add a check in another PR

Reply:
https://github.com/holoviz/hvplot/pull/1144